Modules/signalmodule.c

```diff
@@ -390,6 +390,29 @@ signal_pause_impl(PyObject *module)
 
 #endif
 
+/*[clinic input]
+signal.raise_signal
+
+    signalnum: int
+    /
+
+Send a signal to the executing process.
+[clinic start generated code]*/
+
+static PyObject *
+signal_raise_signal_impl(PyObject *module, int signalnum)
+/*[clinic end generated code: output=e2b014220aa6111d input=e90c0f9a42358de6]*/
+{
+    int err;
+    Py_BEGIN_ALLOW_THREADS
```

Comment:
We'll need the `SUPPRESS_IPH` macros around the `raise` call as well, or else passing an invalid value will cause the process to crash on Windows (indicated by the "invalid parameter handler will be invoked" language [here](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/raise))